test/cog/command/service/data_store_test.exs

```diff
@@ -1,34 +1,30 @@
 defmodule Cog.Command.Service.DataStoreTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case, async: false
```

Comment:
I believe that if don't run ecto in shared mode you can run the tests async. Downside is that you have to explicitly allow the connection for secondary processes that need to access the db. But I don't think we have to worry about that for these tests.